features/aanspreekvorm.feature

```diff
@@ -0,0 +1,659 @@
+      Dan heeft de response een persoon met de volgende 'adressering' gegevens
+      | naam                          | waarde                 |
+      | aanschrijfwijze.aanspreekvorm | De hoogwelgeboren heer |
+
+      Voorbeelden:
+      | predicaat | naamgebruik |
+      | JH        | E           |
+      | JV        | V           |
+      | JH        | N           |
+
+    Abstract Scenario: persoon met geslacht "M", een predicaat en aanduiding naamgebruik "P", heeft een ex-partner
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                        | waarde    |
+      | burgerservicenummer         | 999992934 |
+      | geslachtsaanduiding (04.10) | M         |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                                 | waarde      |
+      | adellijke titel of predicaat (02.20) | <predicaat> |
+      | aanduiding naamgebruik (61.10)       | P           |
+      En de persoon heeft een 'partner' met de volgende 'ontbindingHuwelijkPartnerschap' gegevens
+      | naam                                                         | waarde   |
+      | datum ontbinding huwelijk/geregistreerd partnerschap (07.10) | 20220628 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | aanschrijfwijze.aanspreekvorm   |
+      Dan heeft de response een persoon met een leeg 'adressering' object
+
+      Voorbeelden:
+      | predicaat |
+      | JH        |
+      | JV        |
+      | JH        |
+
+    Abstract Scenario: persoon met geslacht "O", een predicaat en aanduiding naamgebruik "<naamgebruik>", heeft een ex-partner
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                        | waarde    |
+      | burgerservicenummer         | 999992934 |
+      | geslachtsaanduiding (04.10) | O         |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                                 | waarde        |
+      | adellijke titel of predicaat (02.20) | <predicaat>   |
+      | aanduiding naamgebruik (61.10)       | <naamgebruik> |
+      En de persoon heeft een 'partner' met de volgende 'ontbindingHuwelijkPartnerschap' gegevens
+      | naam                                                         | waarde   |
+      | datum ontbinding huwelijk/geregistreerd partnerschap (07.10) | 20220628 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | aanschrijfwijze.aanspreekvorm   |
+      Dan heeft de response een persoon met een leeg 'adressering' object
+
```

Comment:
dit is toch uitsluitend het resultant van geslacht = o?  
Zo ja, is het dan nodig om de andere omstandigheden te noemen?